src/Hubbup.Web/Pages/Index.cshtml

```diff
@@ -1,312 +1,19 @@
 @page "{repoSetName}"
-@model IndexModel
+@inject Microsoft.AspNetCore.Hosting.IHostingEnvironment HostingEnvironment
 @{
-    ViewData["PageTitlePrefix"] = Model.RepoSetName;
-    ViewData["CurrentGroup"] = Model.RepoSetName;
+    var repoSet = RouteData.Values["repoSetName"];
+    ViewData["PageTitlePrefix"] = repoSet;
+    ViewData["CurrentGroup"] = repoSet;
 }
 
 <div class="row issue-list-row">
-    <div class="col-md-2 col-md-push-10">
-        <a asp-controller="IssueList" asp-action="Index" asp-route-repoSet="@Model.RepoSetName">Switch to Legacy View</a>
-    </div>
-
-    <div class="col-md-10 col-md-pull-2">
-        <div class="tab-content">
-            <div class="tab-pane active" id="group-by-assignee">
-                @foreach (var person in Model.People)
-                {
-                    <div class="panel panel-default i-assignee-panel" data-assignee="@person" data-id="@TagBuilder.CreateSanitizedId(person, "-")">
-                    </div>
-                }
-            </div>
+    <div class="col-md-12">
+        <a asp-controller="IssueList" asp-action="Index" asp-route-repoSet="@repoSet">Switch to Legacy View</a>
+        <div class="tab-content" id="app-root" data-repo-set="@repoSet" data-base-url="@Url.Content("~/")" data-environment="@HostingEnvironment.EnvironmentName">
```

Comment:
This is a nice pattern - using the root elem attributes to pass over data from the .NET side. I've more often seen people use global vars, but your approach here is cleaner.